web/social.py

```
@@ -334,3 +334,31 @@ def get_social_stats():
     # Add other social media platforms here as needed
 
     return stats
+
+
+from urllib.parse import quote
```

Ruff (0.8.2)

Module level import not at top of file

(E402)